cypress.config.ts

```diff
@@ -16,21 +22,175 @@ export default defineConfig({
   },
   e2e: {
     setupNodeEvents(on, config) {
+      async function connectDB(query: string) {
+        const dbUrl = new URL(config.env.POSTGRES_URL);
+        const client = new pg.Client({
+          user: dbUrl.username,
+          password: dbUrl.password,
+          host: dbUrl.hostname,
+          database: dbUrl.pathname.slice(1),
+          port: Number(dbUrl.port),
+          ssl: false,
+        });
+        await client.connect();
+        const res = await client.query(query);
+        await client.end();
+        return res.rows;
+      }
+
       on('task', {
-        async connectDB(query: string) {
-          const dbUrl = new URL(config.env.POSTGRES_URL);
-          const client = new pg.Client({
-            user: dbUrl.username,
-            password: dbUrl.password,
-            host: dbUrl.hostname,
-            database: dbUrl.pathname.slice(1),
-            port: Number(dbUrl.port),
-            ssl: false,
+        connectDB,
+        async deleteUser(email = 'test@test.com') {
+          const [user] = await connectDB(`SELECT *
+                                          FROM users
+                                          WHERE email = '${email}';`);
+          if (user) {
+            await connectDB(`
+BEGIN;
+
+DELETE FROM organizations WHERE user_id = '${user.id}';
+DELETE FROM users WHERE id = '${user.id}';
+DELETE FROM supertokens_emailpassword_user_to_tenant WHERE email = '${email}';
```

Comment:
maybe we should add `ON DELETE CASCADE` for this table? otherwise I was unable to recreate a user with same email after deletion him in `users` table